pkg/schema/rtinstance/rtinstance.go

```diff
@@ -0,0 +1,7 @@
+package rtinstance
+
+import (
+	"cuelang.org/go/cue"
+)
+
+var Rt = &cue.Runtime{}
```

Comment:
`rt` was an acceptable name when the variable was unexported, but exporting it - even from `internal` - means it should be a real word.
```suggestion
var Runtime = &cue.Runtime{}
```

(of course, it should change to `CueContext` after the v0.4.0 update)